lib/matplotlib/figure.py

```diff
@@ -1001,6 +1002,142 @@ def add_subplot(self, *args, **kwargs):
         self.stale = True
         return a
 
+    def add_subplots(self, nrows=1, ncols=1, sharex=False, sharey=False,
```

Comment:
but, the name `subplots` has been around for a while now and it is better to not rename functionality if we don't have to.  From the point of view of the user moving from pyplot -> OO you don't want to make them re-learn the names of things.